src/DataSets/DataSets.component.js

```diff
@@ -303,21 +332,37 @@ const DataSets = React.createClass({
 
         const logActions = [
             <FlatButton
-                label={this.getTranslation('close')}
+                label={(this.state.logsPageLast < 0 ? this.tr("logs_no_older") : this.tr("logs_older")) +
```

Comment:
See later comment about how to avoid 404 for non-existing pages. I was thinking we could use the same idea: since we already know the existing pages in the store, we can hide the button if we are sure there is no more data to load.